python/ray/tune/tune.py

```diff
@@ -28,10 +28,21 @@
 from ray.tune.callback import Callback
 from ray.tune.error import TuneError
 from ray.tune.experiment import Experiment, _convert_to_experiment_list
-from ray.tune.experimental.output import (
-    get_air_verbosity,
-    _detect_reporter as _detect_air_reporter,
-)
+
+try:
+    from ray.tune.experimental.output import (
+        get_air_verbosity,
+        _detect_reporter as _detect_air_reporter,
+    )
+except Exception:
+
+    def get_air_verbosity(*args, **kwargs):
+        return None
+
+    def _detect_air_reporter(*args, **kwargs):
+        return None
```

Comment:
Why is this needed?

Reply:
because OrderedDict is not available for 3.6 :(